packages/connect/src/types/device.ts

```diff
@@ -27,6 +27,9 @@ export type UnavailableCapability =
     | 'update-required'
     | 'trezor-connect-outdated';
 
+/**
+ * `{first testnet address}@{device.features.device_id}:{device.instance}`
+ */
```

Comment:
useful to see it like this 

<img width="566" alt="image" src="https://github.com/user-attachments/assets/167ede9d-9b4e-4cd1-8041-c24e29f47c75" />